docs/helm-chart/index.rst

```diff
@@ -155,3 +155,31 @@ To run database migrations with Argo CD automatically, you will need to add:
 This will run database migrations every time there is a ``Sync`` event in Argo CD. While it is not ideal to run the migrations on every sync, it is a trade-off that allows them to be run automatically.
 
 If you use the Celery(Kubernetes)Executor with the built-in Redis, it is recommended that you set up a static Redis password either by supplying ``redis.passwordSecretName`` and ``redis.data.brokerUrlSecretName`` or ``redis.password``.
+
+
+Naming Conventions
+------------------
+
+For new installations it is highly recommended to start using standard naming conventions.
+It is not enabled by default as this may cause unexpected behaviours on existing installations. However you can enable it using ``--set useStandardNaming=true``
+
+.. code-block:: bash
+
+    helm upgrade airflow -n airflow . --set useStandardNaming=true
```

Comment:
```suggestion
It is not enabled by default as this may cause unexpected behaviours on existing installations. However you can enable it using ``useStandardNaming``:

.. code-block:: yaml

    useStandardNaming: true
```

Let's switch to yaml vs helm --set.

Reply:
updated in https://github.com/apache/airflow/pull/31066/commits/a1f03ecc2baf0b78fb5144735fb333e2765d0b02